src/windows/mod.rs

```diff
@@ -0,0 +1,88 @@
+use crate::Error;
+
+use bindings::Windows::Win32::NetworkManagement::IpHelper::{
+    GetAdaptersAddresses, ADDRESS_FAMILY, AF_INET, AF_INET6, AF_UNSPEC,
+    GET_ADAPTERS_ADDRESSES_FLAGS, IP_ADAPTER_ADDRESSES_LH,
+};
+
+use bindings::Windows::Win32::Networking::WinSock::{SOCKADDR_IN, SOCKADDR_IN6};
+use bindings::Windows::Win32::System::Diagnostics::Debug::{ERROR_BUFFER_OVERFLOW, NO_ERROR};
+use libc::{wchar_t, wcslen};
+use memalloc::{allocate, deallocate};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};

```

Comment:
Nitpick but can we have this re arranged like:

```suggestion
use bindings::Windows::Win32::NetworkManagement::IpHelper::{
    GetAdaptersAddresses, ADDRESS_FAMILY, AF_INET, AF_INET6, AF_UNSPEC,
    GET_ADAPTERS_ADDRESSES_FLAGS, IP_ADAPTER_ADDRESSES_LH,
};

use bindings::Windows::Win32::Networking::WinSock::{SOCKADDR_IN, SOCKADDR_IN6};
use bindings::Windows::Win32::System::Diagnostics::Debug::{ERROR_BUFFER_OVERFLOW, NO_ERROR};
use libc::{wchar_t, wcslen};
use memalloc::{allocate, deallocate};
use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};

use crate::Error;
```

I usually arrange _imports_ in the following order:

1. External (`use crate_name`)
2. Crate level (`use crate::`)
3. Super lever (`use super::`)

Reply:
I think vscode auto format on save has its own opinions hahah, but shure, be my guest

Reply:
I set reorder_imports = false so rustfmt don't cramp your style hehe